src/backends/KiboCommerce.js

```diff
@@ -0,0 +1,194 @@
+import { ProductSelectorError } from '../ProductSelectorError';
+
+export const productSearchQuery = /* GraphQL */ `
+  query Search($query: String, $filter: String, $pageSize: Int, $startIndex: Int) {
+    productSearch(query: $query, filter: $filter, pageSize: $pageSize, startIndex: $startIndex) {
+      items {
+        productCode
+        content {
+          productName
+          productImages {
+            imageUrl
+          }
+        }
+      }
+      totalCount
+      pageSize
+      startIndex
+    }
+  }
+`;
+
+export class APIAuthenticationHelper {
+  constructor(clientId, sharedSecret, authHost) {
+    this._clientId = clientId;
+    this._sharedSecret = sharedSecret;
+    this._authHost = authHost;
+    let authTicket = undefined;
+    try {
+      let serializedAuth = window.localStorage.get('kibo-at');
+      if (serializedAuth) {
+        authTicket = JSON.parse(serializedAuth);
+      }
+    } catch (error) {}
+    this.authData = authTicket;
+  }
+
+  _buildFetchOptions(data) {
+    return {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    };
+  }
```

Comment:
All existing code in the repo has an empty line between methods in a class. Prettier doesn't enforce this, maybe it should.

I'll just leave one comment, but it's inconsistent right now.